sql/create_signature_building_charts.sql

```diff
@@ -0,0 +1,232 @@
+      UNION
+      SELECT bbl, '2018-01' AS month, uc2018 AS rentstab_units FROM rentstab_data
+      UNION
+      SELECT bbl, '2020-01' AS month, uc2020 AS rentstab_units FROM rentstab_data
+      UNION
+      SELECT bbl, '2021-01' AS month, uc2021 AS rentstab_units FROM rentstab_data
+      UNION
+      SELECT bbl, '2022-01' AS month, uc2022 AS rentstab_units FROM rentstab_data
+    ),
+
+    dob_jobs_all AS (
+		SELECT 
+			bbl,
+            to_char(filingdate, 'YYYY-MM') AS month, 
+			count(DISTINCT jobfilingnumber) AS dob_jobs
+		FROM dob_now_jobs
+        INNER JOIN signature_unhp_data USING(bbl) 
+		WHERE filingdate >= (CURRENT_DATE - interval '1' year)
+		GROUP BY bbl, month
+		UNION 
+		SELECT 
+			bbl,
+            to_char(prefilingdate, 'YYYY-MM') AS month, 
+			count(DISTINCT job) AS dob_jobs
+		FROM dobjobs
+        INNER JOIN signature_unhp_data USING(bbl) 
+		WHERE prefilingdate >= (CURRENT_DATE - interval '1' year)
+		GROUP BY bbl, month
+	),
+
+	dob_permits_jobs AS (
+		SELECT
+			bbl,
+            month, 
+			sum(dob_jobs)::int as dobpermits_jobs
+		FROM dob_jobs_all
+		GROUP BY bbl, month
+	),
+
+    hpd_erp_charges_all AS (
+		SELECT 
+			c.bbl, 
+			c.omocreatedate AS order_date,
+			sum(i.chargeamount) AS charge_amount
+		FROM hpd_omo_charges AS c
+		LEFT JOIN hpd_omo_invoices AS i USING(omonumber)
+        WHERE c.omocreatedate >= '2010-01-01'
+		GROUP BY bbl, omonumber, c.omocreatedate
+		UNION
+		SELECT 
+			bbl, 
+			hwocreatedate AS order_date, 
+			chargeamount AS charge_amount
+		FROM hpd_hwo_charges
+        WHERE hwocreatedate >= '2010-01-01'
+	), 
+	
+	hpd_erp AS (
+		SELECT
+			bbl,
+            to_char(order_date, 'YYYY-MM') AS month, 
+			sum(charge_amount)::float AS hpderp_charges
+		FROM hpd_erp_charges_all
+        INNER JOIN signature_unhp_data USING(bbl)   
+		GROUP BY bbl, month
+	)
+
+    SELECT 
+        bbl, 
+        month,
+        coalesce(hpdviolations_class_a, 0) AS hpdviolations_class_a,
+        coalesce(hpdviolations_class_b, 0) AS hpdviolations_class_b,
+        coalesce(hpdviolations_class_c, 0) AS hpdviolations_class_c,
+        coalesce(hpdviolations_class_i, 0) AS hpdviolations_class_i,
+        coalesce(hpdviolations_total, 0) AS hpdviolations_total,
+
+        coalesce(hpdcomplaints_emergency, 0) AS hpdcomplaints_emergency,
+        coalesce(hpdcomplaints_nonemergency, 0) AS hpdcomplaints_nonemergency,
+        coalesce(hpdcomplaints_total, 0) AS hpdcomplaints_total,
+
+        -- combine regular dob violations and ecb violations into one grouping called "dobviolations"
+        coalesce(regulardobviolations_total, 0) AS dobviolations_regular,
+        coalesce(ecbviolations_total, 0) AS dobviolations_ecb,
+        coalesce(regulardobviolations_total, 0) + coalesce(ecbviolations_total, 0) AS dobviolations_total,
+
+        CASE 
+            WHEN pluto_latest.unitsres > 10 THEN coalesce(evictions_filed, 0)
```

Comment:
i thought the rent stab unit threshold for publishing eviction data was 6 units?

Reply:
for OCA data we can report for only 11+ unit buildings. The rent stab 6+ units is the laws for which buildings are subject to RS (6+ units and built before 1974)